packages/cf-component-select/src/MenuHeader.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactSelect from 'react-select';
+import { createComponent } from 'cf-style-container';
+
+const MenuHeader = createComponent(
+  ({ theme }) => ({
+    color: '#454545',
```

Comment:
should be a constant (on of `theme.colors`)

Reply:
(all colors should be constants)

Reply:
How are new colors named?